### ISSUE TITLE
Case insensitive replace

### ISSUE THREAD

**Issue by filippovdenis** [repeatedly/fluent-plugin-record-modifier#49]
Hi,
Is it possible to perform a case insensitive replace via <replace> and expression statement?

**Comment by repeatedly** [comment]
You mean `<replace>` feature or something?

https://github.com/repeatedly/fluent-plugin-record-modifier#replace_keys_value

**Comment by jonathanhartley** [comment]
Yes I would also like to do /test StriNg/i  or something similar, I have tried this and it does not work, could a option be added ?
